src/main/java/edu/cornell/kfs/coa/document/validation/impl/CuAccountGlobalRule.java

```diff
@@ -0,0 +1,51 @@
+package edu.cornell.kfs.coa.document.validation.impl;
+
+import java.text.MessageFormat;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.coa.businessobject.AccountGlobalDetail;
+import org.kuali.kfs.coa.document.validation.impl.AccountGlobalRule;
+import org.kuali.kfs.kns.document.MaintenanceDocument;
+
+import edu.cornell.kfs.sys.CUKFSKeyConstants;
+import edu.cornell.kfs.sys.CUKFSParameterKeyConstants;
+import edu.cornell.kfs.sys.CUKFSPropertyConstants;
+
+@SuppressWarnings("deprecation")
+public class CuAccountGlobalRule extends AccountGlobalRule {
+    
+    @Override
+    protected boolean processCustomRouteDocumentBusinessRules(MaintenanceDocument document) {
+        boolean valid = super.processCustomRouteDocumentBusinessRules(document);
+        if (doesAccountGlobalHaveAccountRestrictionCodeAndNoSubFUndGroupCode()) {
+            valid = isTheNewRestrictionCodeValidForEachAccount() && valid;
+        }
+        return valid;
```

Comment:
This if statement could possibly be refactored to 
if (valid && doesAccountGlobalHaveAccountRestrictionCodeAndNoSubFundGroupCode()) {
valid = isTheNewRestrictionCodeValidForEachAccount()
}

Reply:
I want to show these error even if there are pre-existing errors from super.processCustomRouteDocumentBusinessRules, so I don't think should be changed

Reply:
Okay I understand now thanks